torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5232,6 +5232,8 @@ def make_idx(n, m):
     reduces = ('prod', 'mean', 'amin', 'amax')
 
     for shape, include_self, reduce in product(shapes, include_selfs, reduces):
+        if op_info.variant_test_name and op_info.variant_test_name != reduce:
```

Comment:
NIT: You can remove this and just set `reduce = op_info.variant_test_name`

Reply:
Nice catch!

Reply:
Done.